proto/osmosis/tokenfactory/v1beta1/genesis.proto

```diff
@@ -29,4 +29,5 @@ message GenesisDenom {
     (gogoproto.moretags) = "yaml:\"authority_metadata\"",
     (gogoproto.nullable) = false
   ];
+  bool whitelisted = 3 [(gogoproto.moretags) = "yaml:\"whitelisted\""];
```

Comment:
Can't we just store whitelisted denoms in module's params literally as a list of tokens?

It's more convenient for maintenance